drivers/scheduler/k8s/k8s.go

```diff
@@ -578,6 +581,7 @@ func (k *K8s) createNamespace(app *spec.AppSpec, namespace string) (*v1.Namespac
 			map[string]string{
 				"creator": "torpedo",
 				"app":     app.Key,
+				"name":    namespace,
```

Comment:
it's a little odd to put the name of the namespace in the labels.

labels are used to group objects and then access them using those group queries. Putting the name as a label has no additional value as you can already access the object directly with names.

Is your use test to tell autopilot to monitor all applications per test or just one application per test? If it's all applications per test, we can add a testname label.